src/hocon_schema_html.erl

```diff
@@ -26,57 +26,60 @@
 -define(REF_PREFIX_ROOT, "root-").
 -define(REF_PREFIX_STRUCT, "struct-").
 
-gen(Schema, Title) ->
+gen(Schema, Title, File) ->
```

Comment:
If it’s a per schema module desc file. Passing in from here won’t work?
The cache insert will have to be dynamic while traversing the schema tree.

Reply:
Current only one desc file support.
if we support  "per schema module desc file".
We have to declare the name of each desc implicitly.
we need to declare as :
```erlang
?DESC("path/to/desc/file", Id).
```
otherwise, we need to put all desc files in the same dir. and define
```erlang
?DESC("DescFile", Id).
```

Neither of the above 2 solutions is as easy as merging the desc directly into a large file.

Reply:
understood, please document it for the api?